Meter.c

```diff
@@ -520,7 +520,7 @@ static uint16_t GraphMeterMode_makeDetailsMask(const GraphColorComputeState* pre
 
    uint8_t blanksAtStart;
    if (prev->nCellsPainted > 0) {
-      blanksAtStart = (uint8_t)maxBlanks % 8 - blanksAtEnd;
+      blanksAtStart = ((unsigned int)(uint8_t)maxBlanks - blanksAtEnd) % 8;
```

Comment:
Why this double cast?

Reply:
I intend the subtraction to wrap over when needed (no negative remainder). This might not look like the best style, I know.
```c
blanksAtStart = ((unsigned int)(int)maxBlanks - blanksAtEnd) % 8U;
```

Reply:
It seems I cannot avoid two casts here, but I've tried to write the casts in a way that makes the most sense (same compiled code):
```c
blanksAtStart = (uint8_t)((int)maxBlanks - blanksAtEnd) % 8;
```